android/assets/jsons/translations/template.properties

```diff
@@ -113,6 +113,7 @@ Requires [PolicyOrNationalWonder] =
 Cannot be purchased = 
 Can only be purchased = 
 See also = 
+Build units with saved UnitType promotion = 
```

Comment:
I think it should not take valueable screen space in the city screen, having very limited screen space on Android in mind. If it has to be a checkbox in the city screen, keep it *as short as possible* by any means. For example:
`[ ] auto promote`